api/crio/crio.go

```diff
@@ -227,8 +232,25 @@ func RootfsCheckpoint(ctx context.Context, ctrDir, dest, ctrID string, specgen *
 
 	for _, change := range rootFsChanges {
 		fullPath := filepath.Join(tmpRootfsChangesDir, change.Path)
-		if err := os.Chown(fullPath, 0, 0); err != nil {
-			fmt.Printf("failed to change ownership for %s: %s", fullPath, err)
+
```

Comment:
I think a nice big comment here explaining what this is and why we're doing it here would be swell

Reply:
done